lib/sigs/delegated/init.go

```diff
@@ -40,10 +41,19 @@ func (s delegatedSigner) Sign(pk []byte, msg []byte) ([]byte, error) {
 }
 
 func (delegatedSigner) Verify(sig []byte, a address.Address, msg []byte) error {
```

Comment:
IMO, I'd "fix" the signature before we even call this (if possible). It would be nice if we didn't have to care about legacy txns here.